packages/server/package.json

```diff
@@ -96,8 +96,8 @@
     "@apollo/utils.withrequired": "^1.0.0",
     "@graphql-tools/schema": "^9.0.0",
     "@josephg/resolvable": "^1.0.0",
-    "@types/express-serve-static-core": "4.17.30",
-    "@types/express": "4.17.13",
+    "@types/express-serve-static-core": "^4.17.30",
```

Comment:
I think we've had these as non-carets for a while because every so often DT puts out a change to one or the other that breaks if you don't update in lockstep (and they screw up the dependency range between them). But maybe they've slowed down those sorts of breaking changes... it's worth at least trying to be less pinned for a moment.